### PR TITLE
fix tointeger function parsing int64 values

### PR DESCRIPTION
**Pull request by AviAvni**
fix https://github.com/FalkorDB/FalkorDB/issues/687

### CODE REVIEW

File: src/arithmetic/numeric_funcs/numeric_funcs.c [hunk 1]
```diff
@@ -130,6 +130,11 @@ SIValue AR_TOINTEGER(SIValue *argv, int argc, void *private_data) {
 	case T_STRING:
 		if(strlen(arg.stringval) == 0) return SI_NullVal();
 		errno = 0;
+		if(strchr(arg.stringval, '.') == NULL) {
+			uint64_t parsedval = strtoll(arg.stringval, &sEnd, 10);
```
Comment (swilly22) [inline src/arithmetic/numeric_funcs/numeric_funcs.c:134]: `strtoll` returns `long long int` negative numbers included.

File: tests/flow/test_function_calls.py [hunk 2]
```diff
@@ -685,6 +685,7 @@ def test23_toInteger(self):
             """RETURN toInteger('1.9')""": [[1]],
             """RETURN toInteger(true)""": [[1]],
             """RETURN toInteger(false)""": [[0]],
+            """RETURN toInteger('1790460441484152222')""": [[1790460441484152222]],
```
Comment (swilly22) [inline tests/flow/test_function_calls.py:688]: please add a test for negative values

**Review by swilly22 — CHANGES_REQUESTED** [review]

**Review by swilly22 — APPROVED** [review]

**Review by swilly22 — APPROVED** [review]
